plugins/in_tail/tail_config.h

```diff
@@ -83,9 +83,9 @@ struct flb_tail_config {
     int refresh_interval_sec;  /* seconds to re-scan           */
     long refresh_interval_nsec;/* nanoseconds to re-scan       */
     int read_from_head;        /* read new files from head     */
-    int rotate_wait;           /* sec to wait on rotated files */
+    time_t rotate_wait;        /* sec to wait on rotated files */
     int watcher_interval;      /* watcher interval             */
-    int ignore_older;          /* ignore fields older than X seconds        */
+    time_t ignore_older;       /* ignore fields older than X seconds        */
```

Comment:
No, this is not what I meant, I didn't ask you to change more things, I asked you to change less things, please rever this.

Reply:
OK, so I understand that you prefer int over time_t for FLB_CONFIG_MAP_BOOL types inside the plugin config structs, I will update the codes again and test them tomorrow.

Reply:
The thing is it's not a boolean field.